src/app/Repositories/ServerAddressLookupRepository.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Repositories;
+
+use App\Interfaces\ServerAddressLookupRepositoryInterface;
+use Illuminate\Http\Client\HttpClientException;
+use Illuminate\Support\Facades\Http;
+
+class ServerAddressLookupRepository implements ServerAddressLookupRepositoryInterface
+{
+    protected array $urls = [
+        "aHR0cDovL2NoZWNraXAuYW1hem9uYXdzLmNvbQ==",
```

Comment:
Any reason why you are using `http` instead of `https`?  Seems unnecessary to base64 encode.

Reply:
Main use case for obfuscating stuff like this is to not show up in GitHub code searches.

Reply:
totes should be https